recipe/meta.yaml

```diff
@@ -186,10 +161,13 @@ outputs:
         - PyQt5.QtXml
         - PyQt5.QtXmlPatterns
 {% endif %}
+      requires:
+        - pip
       commands:
         # we don't have xvfb on our builders ... so we might ignore it ..
         - DISPLAY=localhost:1.0 xvfb-run -a bash -c 'python pyqt_test.py' || true # [linux]
```

Comment:
{% if enable_testingui %} ? I wonder does it even work on prefect ?

Reply:
enable_testingui is set at the top of the recipe. It is disabled for linux-aarch64 and linux-64 because X11 is not installed on our builders.